code/starfield/starfield.cpp

```diff
@@ -2668,6 +2688,16 @@ void stars_delete_entry_FRED(int index, bool is_a_sun)
 	}
 }
 
+// Goober5000
+void stars_add_blank_background(bool creating_in_fred)
+{
+	Backgrounds.emplace_back();
+
+	// any background that is created will have correct angles, so be sure to set the flag
+	if (creating_in_fred)
+		Backgrounds.back().flags.set(Starfield::Background_Flags::Corrected_angles_in_mission_file);
```

Comment:
If the new checkbox in the Background Editor is unchecked, where does the flag get unset? In `background_flags_close()`?

Reply:
Yes.